dbos/_kafka.py

```diff
@@ -34,6 +43,12 @@ def on_error(err: KafkaError) -> NoReturn:
     if "auto.offset.reset" not in config:
         config["auto.offset.reset"] = "earliest"
 
+    if config.get("group.id") is None:
+        config["group.id"] = safe_group_name(func.__qualname__, topics)
+        dbos_logger.debug(
```

Comment:
This should frankly be a warning. Users should always specify their consumer group.

Reply:
Sure I can change this to warning.